app/__init__.py

```diff
@@ -10,13 +12,11 @@ def create_app(config=None):
     app.config['SQLALCHEMY_DATABASE_URI'] = os.environ.get('SQLALCHEMY_DATABASE_URI')
 
     if config:
-        # Merge `config` into the app's configuration
-        # to override the app's default settings for testing
         app.config.update(config)
 
     db.init_app(app)
     migrate.init_app(app, db)
 
-    # Register Blueprints here
-
+    app.register_blueprint(tasks_bp, url_prefix="/tasks")
+    app.register_blueprint(goals_bp, url_prefix="/goals")
```

Comment:
```suggestion
    app.register_blueprint(tasks_bp)
    app.register_blueprint(goals_bp)
```

We don't need to add `url_prefix` here since we do it in the route files (see `goal_routes.py` line 6 and `task_routes.py` line 8).